File: BAEKJOON/cpp/5586_JOI와_IOI.cpp
#include <iostream>
#include <string>
using namespace std;

string s;
int joi_cnt = 0, ioi_cnt = 0;

void input() {
	cin >> s;
}

void solution() {
	for (int i = 0; i < s.size() - 2; i++) {
		if (s[i + 0] == 'J' && s[i + 1] == 'O' && s[i + 2] == 'I') {
			joi_cnt++;
		}

		if (s[i + 0] == 'I' && s[i + 1] == 'O' && s[i + 2] == 'I') {
			ioi_cnt++;
		}
	}

	cout << joi_cnt << "\n" << ioi_cnt;
}

int main()
{
	ios::sync_with_stdio(false);
	cin.tie(NULL);
	cout.tie(NULL);

	input();
	solution();

	return 0;
}

File: BAEKJOON/java/5586_JOI와_IOI.java
import java.io.*;

public class Main {
    public static String s;
    public static int joiCnt = 0, ioiCnt = 0;

    public static BufferedReader br = new BufferedReader(new InputStreamReader(System.in));
    public static BufferedWriter bw = new BufferedWriter(new OutputStreamWriter(System.out));

    public static void main(String[] args) throws IOException {
        input();
        solution();

        bw.flush();
        bw.close();
    }

    private static void input() throws IOException {
        s = br.readLine();
    }

    private static void solution() throws IOException {
       for (int i = 0; i < s.length() - 2; i++) {
           if (s.charAt(i) == 'J' && s.charAt(i + 1) == 'O' && s.charAt(i + 2) == 'I') {
               joiCnt++;
           }

           if (s.charAt(i) == 'I' && s.charAt(i + 1) == 'O' && s.charAt(i + 2) == 'I') {
               ioiCnt++;
           }
       }

       bw.write(joiCnt + "\n" + ioiCnt);
    }
}